warehouse/models/mart/gtfs/fct_hourly_rt_feed_files.sql

```diff
@@ -0,0 +1,109 @@
+{{ config(materialized='incremental', unique_key = 'key') }}
+
+-- BigQuery does not do partition elimination when using a subquery: https://stackoverflow.com/questions/54135893/using-subquery-for-partitiontime-in-bigquery-does-not-limit-cost
+-- save max date in a variable instead so it can be referenced in incremental logic and still use partition elimination
+{% if is_incremental() %}
+    {% set dates = dbt_utils.get_column_values(table=this, column='dt', order_by = 'dt DESC', max_records = 1) %}
+    {% set max_date = dates[0] %}
+{% endif %}
+
+WITH
+
+int_gtfs_rt__daily_url_index AS (
+    SELECT *
+    FROM {{ ref('int_gtfs_rt__daily_url_index') }}
+),
+
+int_gtfs_rt__unioned_parse_outcomes AS (
+    SELECT *
+    FROM {{ ref('int_gtfs_rt__unioned_parse_outcomes') }}
+),
+
+parse_outcomes AS (
+    SELECT *
+    FROM int_gtfs_rt__unioned_parse_outcomes
+    {% if is_incremental() %}
+    WHERE dt >= DATE '{{ max_date }}'
+    {% else %}
+    WHERE dt >= DATE_SUB(CURRENT_DATE(), INTERVAL 1 MONTH)
+    {% endif %}
+),
+
+daily_tot AS (
+
+    SELECT
+
+        dt,
+        base64_url,
+        feed_type,
+        COUNT(*) as file_count_day
+
+    FROM parse_outcomes
+    GROUP BY
+        dt,
+        base64_url,
+        feed_type
+
+),
+
+hourly_tot AS (
+
+    SELECT *
+    FROM
+        (SELECT
+
+            dt,
+            EXTRACT(HOUR FROM hour AT TIME ZONE "America/Los_Angeles") as download_hour,
```

Comment:
I think we should leave this in UTC for consistency.... We've basically committed to having all system-generated timestamps in UTC (old dashboard is UTC)